code/modules/surgery/wound_repair.dm

```diff
@@ -174,9 +180,8 @@
 		if (target.getToxLoss() >= 0)
 			user.visible_message(SPAN_NOTICE("[user] finishess filtering out any toxins in [target]'s body and repairing any neural degradation with the [tool_name]."), \
 			SPAN_NOTICE("You finish filtering out any toxins to [target]'s body and repairing any neural degradation with the [tool_name].") )
-			if(tool.use(1))
-				target.adjustToxLoss(-200)
-				target.timeofdeath = 99999999
+			target.adjustToxLoss(-200)
+			target.timeofdeath = 99999999
```

Comment:
Why did you make this surgery not consume a stack anymore and didn't mention it in the PR body ?